tests/Routing/RoutingRouteTest.php

```diff
@@ -1378,6 +1380,24 @@ public function testRouteRedirect()
         $this->assertEquals(302, $response->getStatusCode());
     }
 
+    public function testRouteView()
+    {
+        $container = new Container;
+        $factory = m::mock('Illuminate\View\Factory');
+        $factory->shouldReceive('make')->once()->with('pages.contact')->andReturn('Contact us');
+        $router = new Router(new Dispatcher, $container);
+        $container->bind(ViewFactory::class, function () use ($factory) {
+            return $factory;
+        });
+        $container->singleton(Registrar::class, function () use ($router) {
+            return $router;
+        });
+
+        $router->view('contact', 'pages.contact');
+
+        $this->assertEquals('Contact us', $router->dispatch(Request::create('contact', 'GET'))->getContent());
```

Comment:
It's not clear whether you like or dislike my use of `assertEquals` here. Are you suggesting I should do something different?